docs/apache-airflow/faq.rst

```diff
@@ -216,20 +216,35 @@ actually start. If this were not the case, the backfill just would not start.
 What does ``execution_date`` mean?
 ----------------------------------
 
-Airflow was developed as a solution for ETL needs. In the ETL world, you typically summarize data. So, if you want to
-summarize data for 2016-02-19, You would do it at 2016-02-20 midnight UTC, which would be right after all data for
-2016-02-19 becomes available.
-
-This datetime value is available to you as :ref:`Template variables<templates:variables>` with various formats in Jinja templated
-fields. They are also included in the context dictionary given to an Operator's execute function.
+*Execution date* or ``execution_date`` is a historical name for what is called a
+*logical date*, and also usually the start of the data interval represented by a
+DAG run.
+
+Airflow was developed as a solution for ETL needs. In the ETL world, you
+typically summarize data. So, if you want to summarize data for 2016-02-19, You
+would do it at 2016-02-20 midnight UTC, which would be right after all data for
+2016-02-19 becomes available. This interval between midnights of 2016-02-19 and
+2016-02-20 is called the *data interval*, and since the it represents data in
+the date of 2016-02-19, this date is thus called the run's *logical date*, or
+the date that this DAG run is executed for, thus *execution date*.
```

Comment:
```suggestion
typically summarize data. So, if you want to summarize data for ``2016-02-19``, You
would do it at ``2016-02-20`` midnight ``UTC``, which would be right after all data for
``2016-02-19`` becomes available. This interval between midnights of ``2016-02-19`` and
``2016-02-20`` is called the *data interval*, and since it represents data in
the date of ``2016-02-19``, this date is thus called the run's *logical date*, or
the date that this DAG run is executed for, thus *execution date*.
```

Reply:
Not sure about making UTC inline code; added ticks around date.